Loisrp update getstart (#134)
interledger/interledger.github.io: _content/setup-wallets-commandline.md

## Interledger Protocol (ILP) TestNet: Getting Started

> This tutorial describes how to use the command line and test XRP to:

Reviewer: I tend to prefer the wording "programmatic" or "using an API" since while this tutorial interacts with the API using `curl` on the command-line, what it's really demonstrating is how to interact with Interledger programmatically, and developers may want to adapt these API calls in their own language if they're building an app.

1. Create an account in the Xpring.io Interledger Testnet.

> 2. Add test XRP to your account using the TestNet Rainmaker \(our version of a "faucet"\).

Reviewer: > (our version of a "faucet"\)

I think should be in the 3rd person, since otherwise the perspective might assumed to be Xpring (which isn't preferable).

3. Check your account balance.
4. Send test XRP to another payment pointer.
5. Receive test XRP.

To do the same with a GUI-based interface, refer to [setup-wallets.md].

### 1. Create an account

> Create a new account with this command:

Reviewer: Would it be worth making articulating this in a little more generic way? e.g. "Send an HTTP request to create a new account. For this tutorial, we'll use `curl` on the command line to make requests to the API, but you can use an HTTP client library in any language. Here's how to create an account on the command line:"

```

> > curl --location --request POST 'https://hermes-rest.ilpv4.dev/accounts' \

Reviewer 2: Hopefully we'll have new URLs here in the next day or two. @LoisRP can you check with me before merging this?

Author: Will do. Incidentally, I don't have access to this repo: https://github.com/xpring-eng/hermes-ilp/ .

Author: Confirmed we keep the same URL for now (until a week or two from now).

--header 'Content-Type: application/json' \
--header 'Accept: application/json' \
--data-raw '{
  "assetCode": "XRP",
  "assetScale": "6"
}'
```

> This request returns a payload that contains your `accountId`, an auth token which you will then use to authenticate to the connector, as well as a [Payment Pointer](https://paymentpointers.org/) that can be used to receive value in your test account.

Reviewer 2: ```suggestion
This request returns a payload that contains your `accountId` in addition to an auth token which you will then use to authenticate to the connector. The response also includes a [Payment Pointer](https://paymentpointers.org/) that can be used to receive value in your test account.
```

> This request returns a payload that contains your `accountId`, an auth token which you will then use to authenticate to the connector, as well as a [Payment Pointer](https://paymentpointers.org/) that can be used to receive value in your test account.

Author: Changed to bulleted list.

> Here's an example response payload for reference. Keep track of the generated `accountId` at `accountId`, and your generated auth token at `"customSettings" -> "ilpOverHttp.incoming.simple.auth_token", for use in subsequent steps.

Author: "customSettings" -> "ilpOverHttp.incoming.simple.auth_token"
Any objection to styling this as: `customSettings.ilpOverHttp.incoming.simple.auth_token`  ?

Reviewer 2: I think we should keep this notation as-is (i.e., maintain the arrow).

### 2. Add test XRP to your account

The Testnet has a rainmaker account that you can use to populate your account with test XRP, so you can then use this account to send test XRP to other accounts. To add test XRP to your account, use the following command. Make sure to replace `{your-account-id}` with your own `accountId` created in Step 1.

```
> curl --location --request POST \
  'https://hermes-rest.ilpv4.dev/accounts/{your-account-id}/money'
```

### 3. Check your account balance

To see how much money is in your account, try the following call. Make sure to replace **`{auth_token}`** above with the auth token returned when you generated your account.

```
> curl --location --request GET 'https://hermes-rest.ilpv4.dev/accounts/{your-account-id}/balance' \
--header 'Accept: application/json' \
--header 'Authorization: Bearer {auth_token}' \
```

This request will return a JSON response similar to the following:

```text
{
    "assetCode": "XRP",
    "assetScale": "6",
    "accountBalance": {
        "accountId": "user_qqGV8nij",
        "netBalance": "0",
        "clearingBalance": "0",
        "prepaidAmount": "0"
    }
}
```

### 4. Send test XRP to another payment pointer

> You can use your account to send test XRP to a different payment pointer. This example uses a destination payment pointer on [https://rafiki.money](https://rafiki.money). Substitute your `accountId` from Step 1. You can change the destination payment pointer if you prefer.

Reviewer: I really like the explanation/replacing the accountId and auth tokens throughout the tutorial. Very clear 👍

```
> curl --location --request POST 'https://hermes-rest.ilpv4.dev/accounts/{your-account-id}/pay' \
--header 'Content-Type: application/json' \
--header 'Accept: application/json' \
--header 'Authorization: Bearer {auth_token}' \
--data-raw '{
  "amount": "10",
  "destinationPaymentPointer": "$rafiki.money/p/dfuelling"
}'
```

This request returns a JSON response similar to the following:

```text
{
    "originalAmount": "10",
    "amountDelivered": "10",
    "amountSent": "10",
    "successfulPayment": true
}
```

- **`originalAmount`** is the amount that you wanted to send.

> - **`amountDelivered`** is the amount your friend actually received.

Reviewer: Might be worth noting that (I believe) this is denominated in the recipient's asset and units, whereas "amount sent" is in the asset and units of the sender -- these can be different
